src/components/Card/index.jsx

```diff
@@ -0,0 +1,45 @@
+import { useCallback } from 'react';
+import styles from './card.module.css';
+import PropTypes from 'prop-types';
+import beautifyDate from '@/utils/beautifyDate';
+
+const Card = ({ link }) => {
+  const [beautifiedDate, beautifiedTimeDiff] = beautifyDate(link.createdAt);
+
+  const handleNavigateCardLink = useCallback(
+    (e) => {
+      if (e.target.closest('.cardAsterisk')) return;
+      window.open(link.url);
+    },
+    [link.url],
+  );
+
+  return (
+    <div className={`${styles.card}`} onClick={handleNavigateCardLink}>
+      <div className={`${styles.cardAsterisk}`}>
+        <img className={`cardAsterisk`} src={'/assets/card-asterisk.svg'} alt="Card Asterisk" />
```

Comment:
```suggestion
        <img className="cardAsterisk" src="/assets/card-asterisk.svg" alt="Card Asterisk" />
```
단순 string을 prop으로 전달할 땐 이런식으로 통일하는게 좋을 것 같아요.